### PR TITLE
[StyleCop] Fix all StyleCop warnings in FrenchDateTimePeriodExtractorConfiguration

### PR DESCRIPTION
**Pull request by JuanAr**
- Reordered fields, properties and methods. Public before private and static before non-static.
- Removed regions
- Added spaces when needed

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDateTimePeriodExtractorConfiguration.cs [hunk 1]
```diff
@@ -7,47 +7,8 @@ namespace Microsoft.Recognizers.Text.DateTime.French
...
-
-        private static readonly Regex TimeTimeUnitRegex =
-            new Regex(DateTimeDefinitions.TimeUnitRegex, RegexOptions.Singleline);
-
-        private static readonly Regex TimeFollowedUnit =
-            new Regex(DateTimeDefinitions.TimeFollowedUnit, RegexOptions.Singleline);
-
         public static readonly Regex TimeNumberCombinedWithUnit =
-            new Regex(DateTimeDefinitions.TimeNumberCombinedWithUnit, RegexOptions.Singleline);
+    new Regex(DateTimeDefinitions.TimeNumberCombinedWithUnit, RegexOptions.Singleline);
```
Comment (guom08) [inline .NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDateTimePeriodExtractorConfiguration.cs]: NIT: Indentation. Please revert this change in the following PR if you can, thanks.

Comment (JuanAr) [reply .NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDateTimePeriodExtractorConfiguration.cs]: Done in https://github.com/Microsoft/Recognizers-Text/pull/1116.
Thanks for the feedback!
